crypto/storage/vault.go

```diff
@@ -67,6 +67,20 @@ type vaultKVStorage struct {
 	client logicaler
 }
 
+func (v vaultKVStorage) Name() string {
+	return VaultConfigKey
+}
+
+func (v vaultKVStorage) CheckHealth() map[string]core.Health {
+	health := make(map[string]core.Health)
+	if err := v.checkConnection(); err != nil {
```

Comment:
what is the time-out of `checkConnection`, in other words, how long can it block calls to the health check endpoint?

Reply:
It uses `VaultConfig.Timeout` which is initialized in `vault.DefaultVaultConfig` and set to 5 seconds.